cypress/integration/search/actions.spec.ts

```diff
@@ -3,6 +3,9 @@
 describe('Start', () => {
   beforeEach(() => {
     cy.visit(Cypress.config().baseUrl!);
+    // seems we could run into race condition
+    // if we don't wait for .DocSearch-Button-Key to render
+    cy.get('.DocSearch-Button-Key');
```

Comment:
I found the tests became flaky without this.

Reply:
Ah great catch!